WifiART/src/userinterface/ArtMvcController.java

```diff
@@ -26,5 +42,5 @@ void modelCallback() {
         fv.setViewState(fm.getState());
     }
 
-    //Listener classes:
+    // Listener classes:
```

Comment:
This can be removed?

Reply:
I'm leaving it in for now because I'm going to be adding listener class definitions there. Technically it can be removed because nothing's there at the moment, but I put some comments just to plan out the structure of the Controller.